### ISSUE TITLE
Backlight font icons not showing correctly

### ISSUE THREAD

**Issue by LorenzoBettini** [Alexays/Waybar#2192]
I've just started using Waybar in Hyprland, and I've installed `otf-font-awesome`. Icon fonts show correctly, but the ones concerning brightness:

![20230529_18h09m22s_grim](https://github.com/Alexays/Waybar/assets/1202254/5ccc1c52-385b-4590-a354-1a86984684bb)

I haven't touched the style.css, which is the default one.

Am I missing something?

**Comment by fleminra** [comment]
tl;dr: try this instead:
```
        "format-icons": ["🌑", "🌘", "🌗", "🌖", "🌕"]
```

These icons were changed in this commit: https://github.com/Alexays/Waybar/commit/25536f70a44f5b3e5829c31037a1ddb4708f7622, with the comment "use moon phases for backlight format-icons."

The symbols in this commit are various characters between U+E38D and U+E3D3. Those code points are in the Unicode "Private Use" block. The Private Use block is where Font Awesome put a lot of (all of?) its symbols. But in my version of Font Awesome (Debian 12's 5.0.10+really4.7.0~dfsg-4.1), there is nothing at these code points.

I have not yet found which other font might have moon-phase symbols at those code points.

However, a lot of these moon-phases were approved as part of the Unicode 6.0 standard. So it seems to me it would be better to use those, than these other Private Use symbols from some unknown font. So that's what my above suggestion is.

**Comment by LorenzoBettini** [comment]
> tl;dr: try this instead:
> 
> ```
>         "format-icons": ["🌑", "🌘", "🌗", "🌖", "🌕"]
> ```
> 

I'll try them, thanks!

> I have not yet found which other font might have moon-phase symbols at those code points.

Sorry, I've always forgotten to update this issue: this `ttf-arimo-nerd` provides the missing icons for moon-phase symbols at those code points (of course, you have to install `otf-font-awesome` as well for most of the other icons).

> 
> However, a lot of these moon-phases were approved as part of the Unicode 6.0 standard. So it seems to me it would be better to use those, than these other Private Use symbols from some unknown font. So that's what my above suggestion is.

I agree we should not rely on private use symbols :)

**Comment by claman** [comment]
Thanks very much @fleminra ! This also resolved my issue.

**Comment by OisinHick** [comment]
This fixed the issue for me, thank you!

**Comment by Alexays** [comment]
Closing — this was resolved/answered in the thread (config, environment, or packaging — not a Waybar bug still open). Feel free to reopen if you're still stuck. 🙏

**Comment by sudg6** [comment]
You need to install  `ttf-jetbrains-mono-nerd` then,
```
fc-cache -f -v
```
